web/beacon-app/src/components/MenuDropdown/OrganizationMenuDropdown.tsx

```diff
@@ -7,21 +7,23 @@ interface RadixDropdownMenuProps {
   items: any;
   trigger?: React.ReactNode;
   isOpen?: boolean;
+  'data-cy'?: string;
   onOpenChange?: (isOpen: boolean) => void;
 }
 
 const OrganizationMenuDropdown = ({
   items,
   trigger,
   isOpen,
+  'data-cy': dataCy,
   onOpenChange,
 }: RadixDropdownMenuProps) => {
   //console.log('items menu', items);
   return (
     <div className="relative mt-4">
       <DropdownMenuPrimitive.Root open={isOpen} onOpenChange={onOpenChange}>
-        <DropdownMenuPrimitive.Trigger>
```

Comment:
`DropdownMenuPrimitive.Trigger` appears as a `button` in the DOM, so the other `button` was removed since we don't need a button inside of another button.